src/editor/node_marker.cpp

```diff
@@ -78,13 +76,55 @@ NodeMarker::get_settings()
 {
   ObjectSettings result(_("Path Node"));
   result.add_float(_("Time"), &(m_node->time));
+  result.add_float(_("Speed"), &(m_node->speed));
   return result;
 }
 
 void
 NodeMarker::editor_update()
 {
   set_pos(m_node->position - Vector(8, 8));
+  update_node_time(m_node, next_node());
+}
+
+std::vector<Path::Node>::iterator NodeMarker::prev_node() {
```

Comment:
For me type name `std::vector<Path::Node>::iterator` seems a bit long and hard to read. It could be my lack of experience in bigger projects. Also I'm not sure, what is our policy regarding naming types.

Reply:
We have a couple typedef's lying around the code, so we definitely use them.